packages/search-types/CHANGELOG.md

```diff
@@ -1,24 +1,16 @@
 # Change Log
 
-All notable changes to this project will be documented in this file.
-See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+All notable changes to this project will be documented in this file. See
```

Comment:
al the changelog changes are done by prettier or by hand?

Reply:
These ones (`-`) are done by Prettier. The ones with `*` are generated by lerna. I don't know why some times the changelog is formatted and some times not. I've created a task to dig deeper into it.